recommender/beta/cloud-client/src/test/java/com/example/recommender/ListRecommendationsTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.recommender;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ListRecommendationsTest {
+
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+
+  @Before
+  public void setUp() throws Exception {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+  }
+
+  @After
+  public void tearDown() {
+    System.setOut(null);
+  }
+
+  @Test
+  public void listRecommendations() throws IOException {
+    ListRecommendations.listRecommendations();
```

Comment:
You'll need to update to `listRecommendations(String projectId, String location, String recommender)` - When move the env vars to this side, make sure to throw an error if they aren't there. See [this example](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/dlp/src/test/java/dlp/snippets/InspectTests.java#L49).

Reply:
Updated and added env var checks.